internal/bind/params.go

```diff
@@ -233,3 +220,33 @@ func Params(args ...interface{}) (parameters []*params.Parameter, _ error) {
 
 	return parameters, nil
 }
+
+func handleNamedValue(arg driver.NamedValue, i, argsLen int) ([]*params.Parameter, error) {
```

Comment:
1. The function receive and return exactly one parameter, fix return value from `[]*params.Parameter` to `*params.Parameter`
2. Better function name: paramHandleNamedValue, with main function prefix. It is better understand about it helper function only.
3. It needs better name for argument `i`, for example `paramNumber`, because `i` hasn't counter semantic as argument.